src/AdminConsole/Pages/Organization/Verify.cshtml

```diff
@@ -6,16 +6,19 @@
     ViewData["Title"] = "Verify your account";
 }
 
-<p>We've sent a confirmation link to the email address you provided. Click on the link in the email to verify your account</p>
+@section Head {
+    <!-- This to refresh the page to check if the user has verified and logged in -->
```

Comment:
You could use a C# comment (`//`) to make this invisible to the users. In some places you need to use (`@* ... *@`) for server-side comments, but since it's in a section, `//` should work.